docs/training.rst

```diff
@@ -256,8 +280,8 @@ A sane authoring process is a feedback loop something like this:
 1. Collect samples. Observe patterns in the :term:`target` nodes as you do.
 2. Write a few rules based on your observations.
 3. Run the trainer. Start with 10-20 training pages and an equal number of validation ones.
-4. If accuracy is insufficient, examine the failing pages. FathomFox's Evaluator page is invaluable for this, as it will show you the element Fathom spuriously picked. Remediate by changing or adding rules. If there are smells Fathom is missing—positive or negative—add rules that score based on them.
+4. If accuracy is insufficient, examine the failing training pages. The trainer will point these out on the commandline, but FathomFox's Evaluator will help you track down ones that are hard to distinguish from their excerpts. Remediate by changing or adding rules. If there are smells Fathom is missing—positive or negative—add rules that score based on them.
```

Comment:
I think "...from their excerpts" is not clear since there is no mention of the partial tag printouts in the trainer output.

Reply:
Fixed; thank you!